ilc/server.js

```diff
@@ -7,21 +10,40 @@ const serveStatic = require('./serveStatic');
 
 app.get('/ping', (req, res) => res.send('pong'));
 
-
 const tailor = tailorFactory(config.get('registry.address'), config.get('cdnUrl'));
 tailor.on('error', (req, err) => {
     console.error('Tailor error:');
     console.error(err);
 });
 
+app.set('view engine', 'ejs');
```

Comment:
Not sure that we need this. We can just use `ejs` lib in the code below.

Reply:
Updated